index.js

```diff
@@ -677,7 +729,7 @@ const autoScript = {
   validateZones: ['createZones', function (results, cb) {
     overallProgress.beginTask('Validating timezone boundaries')
     loadDistZonesIntoMemory()
-    if (process.argv.indexOf('no-validation') > -1) {
+    if (argv.no_validation) {
```

Comment:
I can't comment on the line directly, but the `zipInputData` function still has some hardcoded values that should probably be changed to the folders received from yargs parsing the cli input.

Reply:
Done.